### ISSUE TITLE
Error parsing

### ISSUE THREAD

**Issue by olegs** [hsz/idea-latex#9]
I have the following code, and `\begin{figure}` is highlighted as unexpected token.

```
% Command for centering image, keeping aspect ratio
\newcommand*{\includeimage}[2][]{%
    \centering{
        \begin{figure}
            \includegraphics[width=\textwidth, height=0.8\paperheight, keepaspectratio]{#2}
        \end{figure}
    }
}
```

**Comment by hsz** [comment]
@olegs Thank you for reporting. Issue has been fixed and will be available with the new `v0.3` release.

**Comment by olegs** [comment]
@hsz Thanks, looking forward for the new releases!
